pkg/helm/utils.go

```diff
@@ -49,18 +51,23 @@ type HelmConfig struct {
 }
 
 func CreateHelmConfig(kubeContext, namespace string, logger *logrus.Logger) (*HelmConfig, error) {
+	tlLog.SendCloudReport("info", "install-script-create-helm", nil)
```

Comment:
Move this to Install function on the helm

Reply:
Updated